components/Search/SearchInput/SearchInput.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import classNames from "classnames";
+
+import style from "./SearchInput.module.css";
+import { SearchSuggestionBlock } from "../SearchSuggestionBlock/SearchSuggestionBlock";
+
+const debounce = (func, timeout = 300) => {
+  let timer;
+  return (...args) => {
+    clearTimeout(timer);
+    timer = setTimeout(() => {
+      func.apply(this, args);
+    }, timeout);
+  };
+};
+
+export const SearchInput = ({ isVisible }) => {
+  const [suggestions, setSuggestions] = React.useState([]);
+  const styleInputBlock = classNames(style.searchInput__container, {
+    [style.hidden]: !isVisible
```

Comment:
Let's make the `hidden={!isVisible}` attribute. It will be great for accessibility